lib/reports/subscriber_lists_report.rb

```diff
@@ -0,0 +1,70 @@
+class Reports::SubscriberListsReport
+  attr_reader :start_of_day, :end_of_day
+
+  CSV_HEADERS = %i[title
+                   slug
+                   matching_criteria
+                   created_at
+                   individual_subscribers
+                   daily_subscribers
+                   weekly_subscribers
+                   individual_updates_for_date
+                   unsubscribed].freeze
+
+  def initialize(date)
+    @start_of_day = Time.zone.parse(date)
+    @end_of_day = Time.zone.parse(date).end_of_day
+  end
+
+  def call
+    non_empty_lists = SubscriberList.where(
+      Subscription.active_on(end_of_day)
+                  .where("subscriber_lists.id = subscriptions.subscriber_list_id")
+                  .arel.exists,
+    )
+
+    CSV.instance($stdout, headers: CSV_HEADERS, write_headers: true) do |csv|
```

Comment:
Minor: I think it's more conventional to use `.generate`.

Reply:
`.generate` doesn't actually output to stdout (it would using puts) but it turns out using `CSV do ...` is suffice.